doc/user-stories/expert-partitioner.md

```diff
@@ -680,10 +680,7 @@ The specs are based on current version of expert partitioner for TW. The goal is
       * 'File System Id' is not shown
     * set encrypt password (optional)
       * same wizard step than in partition creation (see above)
-* *(pending) and the row selected is a thin pool lv*
```

Comment:
Because the spec was wrong. The fact that a thin pool is selected is not enough to automatically propose a new thin volume. Thin volume is automatically when it is only possible to create a thin volume (no available space in the VG and a thin pool exists).

Reply:
Ah, ok. Fine then.